### DISCUSSION TITLE
macaddress.io

### DISCUSSION THREAD

**Discussion by eric-holtzclaw** [Security-Onion-Solutions/securityonion#7304] [Ideas] [answered]
Can I request this python client to get included and mapped to MAC addresses in Kibana?
https://github.com/CodeLineFi/maclookup-python

**Comment by dougburks** [comment, marked as answer]
Sorry, we have no plans to include this.

Please keep in mind that for most enterprise deployments, MAC addresses seen by sensors are only going to be for the next hop and not the actual endpoint.

If you really want to look up MAC addresses, you might be able to add a new option to your Hunt action menu:
https://docs.securityonion.net/en/2.3/soc-customization.html#action-menu
